lib/moonshot/controller_config.rb

```diff
@@ -21,5 +25,17 @@ def initialize
       @plugins = []
       @show_all_stack_events = false
     end
+
+    def parameter_strategy=(value)
```

Comment:
In initialize for ControllerConfig, set a new instance of the DefaultStrategy class, then you only have to override it when it's non-default and the controller is functional without CLI.